lib/jni/PrivacyGuardState.hpp

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (c) 2015-2020 Microsoft Corporation and Contributors.
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <jni.h>
+
+#include "modules/privacyguard/CommonDataContext.hpp"
+#include "modules/privacyguard/PrivacyGuard.hpp"
+#include "ctmacros.hpp"
+
+namespace MAT_NS_BEGIN
+{
+
+class PrivacyGuardState
```

Comment:
Why bother making this a class if there's no instances nor state of said instances?

Reply:
No reason at all to have this as a class now that I think of it. Updated.